extensions/amp-story/0.1/amp-story.js

```diff
@@ -1433,6 +1450,21 @@ export class AmpStory extends AMP.BaseElement {
     this.switchTo_(dev().assertElement(this.pages_[0].element).id);
   }
 
+  /** @return {!NavigationState} */
```

Comment:
Per comments above, I'm not sure this all needs to be exposed; I think you can instead add `insertPage` and access internals here instead.

Reply:
this one was left for now